doc/3/controllers/collection/update-specifications/index.md

```diff
@@ -0,0 +1,52 @@
+---
+code: true
+type: page
+title: updateSpecifications
+description: Update the validation specifications
+---
+
+# updateSpecifications
+
+The updateSpecifications method allows you to create or update the validation specifications for a collection.
+
+When the validation specification is not formatted correctly, a detailed error message is returned to help you to debug.
+
+<br/>
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> updateSpecifications(
+      final String index,
+      final String collection,
+      final ConcurrentHashMap<String, Object> specifications)
+```
+
+<br/>
+
+| Arguments        | Type                                         | Description              |
+| ---------------- | -------------------------------------------- | ------------------------ |
+| `index`          | <pre>String</pre>                            | Index name               |
+| `collection`     | <pre>String</pre>                            | Collection name          |
+| `specifications` | <pre>ConcurrentHashMap<String, Object></pre> | Specifications to update |
+
+### specifications
+
+A `ConcurrentHashMap<String, Object>` representing the specifications.
+
+It must follow the [Specification Structure](/core/2/guides/essentials/data-validation):
+
+```java
```

Comment:
The content of this example doesn't appear to be written in java.

Given that there is already a code snippet in java, and JSON examples in the link you put a few lines back, then I think that this example should be removed.

Reply:
I think it's the output of a `println` on the `HashMap` but still I agree with @scottinet, you should keep the JSON